CMakeLists.txt

```diff
@@ -312,7 +312,7 @@ find_package(SDL2_ttf REQUIRED)
 if(TARGET SDL2_ttf::SDL2_ttf)
 	add_library(SDL2::TTF ALIAS SDL2_ttf::SDL2_ttf)
 endif()
-find_package(TBB REQUIRED)
+find_package(TBB)
```

Comment:
Instead of silently failing in case of missing TBB (and not building NullkillerAI) can you add option? 
Something like
`option(ENABLE_NULLKILLER_AI "Enable compilation of nullkiller AI library" ON)`
And if this option is off - don't try looking for TBB and don't include nullkiller subdirectory